leaderrotation/roundrobin.go

```diff
@@ -30,5 +32,6 @@ func NewRoundRobin() modules.LeaderRotation {
 }
 
 func chooseRoundRobin(view hotstuff.View, numReplicas int) hotstuff.ID {
-	return hotstuff.ID(view%hotstuff.View(numReplicas) + 1)
+	id := uint64(math.Ceil(float64(view) / 2.0))
```

Comment:
I didn't understand this change. Is it related to the latency? 

Was to original version wrong/problematic? Maybe add an explainer comment here.